### PR TITLE
chore : add mocks folder for all test data

### PR DESCRIPTION
**Pull request by rishusingh022**

### CODE REVIEW

File: tests/__mocks__/auth.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const login = {
+  mockRes: {
+    json: jest.fn(),
+    status: jest.fn().mockReturnThis(),
+  },
+  mockReq: {
+    body: {
+      email: 'promit.revar2211@gmail.com',
```
Comment (rohini-senthil) [inline tests/__mocks__/auth.js:8]: Can we create a variable called testEmail with value johndoe@mckinsey.com and use it in this file wherever needed. Similarly with other values that are repeated more than once in this file

File: tests/__mocks__/case-study.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  resolvedValue: {
+    case_study_id: '1',
+    name: 'test',
+    description: 'test',
+    collaborators_ids: ['1', '2'],
+    image: 'test',
+    box_link: 'test',
+    engagement_id: '1',
+    createdAt: '2022-01-17T04:33:12.000Z',
+    updatedAt: '2022-01-17T04:33:12.000Z',
```
Comment (rohini-senthil) [inline tests/__mocks__/case-study.js:25]: This whole value is repeated again in toDelete can we declare a variable for it

File: tests/__mocks__/user.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    email: 'johndoe@mckinsey.com',
+    fmno: '123456',
+    current_engagement_ids: ['1', '2'],
+    case_study_ids: ['1', '2'],
+    skills: ['node, react'],
+    role: 'intern',
+    guild: 'swe',
+    past_engagement_ids: ['1', '2'],
+    createdAt: '2022-01-17T04:33:12.000Z',
+    updatedAt: '2022-01-17T04:33:12.000Z',
```
Comment (rohini-senthil) [inline tests/__mocks__/user.js:66]: Value is repeated multiple times, let's declare this as a variable in the file

File: tests/__mocks__/user.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    body: {
+      name: 'Promit Revar',
+    },
+  },
+  resolvedValue: {
+    data: {
+      user_id: '9a492c13-85e8-4b26-9339-a5d037664d1a',
+      email: 'promit.revar2211@gmail.com',
+      name: 'Promit Revar',
+      updatedAt: '2023-02-09T15:02:53.658Z',
```
Comment (rohini-senthil) [inline tests/__mocks__/user.js:22]: Let's change this to a generic email with john doe as well

File: tests/__mocks__/utils.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    body: {
+      email: '',
+      password: 'test',
+    },
+  },
+  resolvedValue: {
+    data: {
+      user_id: '9a492c13-85e8-4b26-9339-a5d037664d1a',
+      name: 'Promit Revar',
+      email: 'promit.revar2211@gmail.com',
```
Comment (rohini-senthil) [inline tests/__mocks__/utils.js:25]: Let's change this to a generic email and name with john doe

File: tests/__mocks__/auth.js (hunk 1 above)
Comment (rishusingh022) [reply tests/__mocks__/auth.js:8]: done !

File: tests/__mocks__/case-study.js (hunk 2 above)
Comment (rishusingh022) [reply tests/__mocks__/case-study.js:25]: done !!

File: tests/__mocks__/user.js (hunk 3 above)
Comment (rishusingh022) [reply tests/__mocks__/user.js:66]: done !!

File: tests/__mocks__/user.js (hunk 4 above)
Comment (rishusingh022) [reply tests/__mocks__/user.js:22]: done !!

File: tests/__mocks__/utils.js (hunk 5 above)
Comment (rishusingh022) [reply tests/__mocks__/utils.js:25]: done !!

**Review by rohini-senthil — APPROVED** [review]

**Pull request merged**